src/components/Details/Legislation/VoteBar.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, useState } from "react";
+import styled from "@emotion/styled";
+import { useMediaQuery } from "react-responsive";
+
+import { VOTE_DECISION } from "../../../models/constants";
+import { screenWidths } from "../../../styles/mediaBreakpoints";
+import Vote from "../../../models/Vote";
+import { Link } from "react-router-dom";
+import { compact } from "lodash";
+
+const ColorBar = styled.div<{ width: string; zIndex: number; height: number }>((props) => ({
+  position: "absolute",
+  top: 0,
+  left: 0,
+  height: props.height,
+  width: props.width,
+  borderTopLeftRadius: "50em",
+  borderBottomLeftRadius: "50em",
+  borderTopRightRadius: "50em",
+  borderBottomRightRadius: "50em",
+  display: "flex",
+  flex: 1,
+  justifyContent: "space-between",
+  alignItems: "center",
+  zIndex: props.zIndex,
+}));
+
+const VOTE_DECISION_COLOR = {
+  [VOTE_DECISION.APPROVE]: "cdp-bg-acceptance-green",
+  [VOTE_DECISION.ABSTAIN_APPROVE]: "cdp-bg-acceptance-green",
+  [VOTE_DECISION.ABSENT_APPROVE]: "cdp-bg-acceptance-green",
```

Comment:
these triple bars with the same colors may get a bit confusing but I get it for now.

Reply:
There will never be 3 green bars, only 1. What this does is merge the 3 "approve" type votes into 1 green bar.

Reply:
I clarified the code to ensure no confusion. A simple color is passed in.